e2e/passkey/passkeyProxyV2.ethereum.test.ts

```diff
@@ -28,11 +22,13 @@ describe('Passkey Pallet Proxy V2 Ethereum Tests', function () {
     });
 
     it('should transfer via passkeys with root sr25519 key into an ethereum style account', async function () {
+      const startingBalance = (await ExtrinsicHelper.getAccountInfo(receiverKeys)).data.free.toBigInt();
       const accountPKey = getUnifiedPublicKey(fundedSr25519Keys);
       const nonce = await getNonce(fundedSr25519Keys);
+      const transferAmount = 55_000_000n;
```

Comment:
Do we have constants for these like DOLLARS and CENTS?

Reply:
Not sure, but the amount isn't important anyway; we just want to be sure to check that whatever amount is transferred, the balance changes by that amount.

Reply:
But that does remind me, it would be good to also check that the correct sender's balance decreases by the appropriate amount.